### PR TITLE
Minor improvements for Russian translation

### PR DESCRIPTION
**Pull request by raven-kg**

### CODE REVIEW

File: app/lang/ru.json [hunk 1]
```diff
@@ -255,10 +256,10 @@
   "Gmail bandwidth exceeded. Please try again later.": "Превышена пропускная способность Gmail. Пожалуйста, повторите попытку позже.",
   "Go Back": "Назад",
   "Go further with Mailspring Pro": "Зайдите дальше с Mailspring Pro",
-  "Go to %@": "Перейти к %@",
-  "Got it!": "Получить",
-  "Gravatar Profile Photo": "Gravatar фото профиля",
-  "Handle it later!": "Разобраться с нему позже!",
+  "Go to %@": "Перейти в %@",
+  "Got it!": "Ясно!",
+  "Gravatar Profile Photo": "Аватара из Gravatar",
```
Comment (ForNeVeR) [inline app/lang/ru.json]: [wiktionary](https://ru.wiktionary.org/wiki/%D0%B0%D0%B2%D0%B0%D1%82%D0%B0%D1%80) tells us that the «аватар» word has a male linguistic gender. Could you please update this accordingly?

File: app/lang/ru.json [hunk 2]
```diff
@@ -912,11 +914,11 @@
   "Privacy note: text below will be sent to an online translation service.": "Примечание о конфиденциальности: текст ниже будет отправлен в службу онлайн-перевода.",
   "Reset translation settings": "Сбросить настройки перевода",
   "Restrict width of messages to maximize readability": "Ограничьте ширину сообщений, чтобы максимизировать читабельность",
-  "Stop translating %@": "Хватит переводить %@",
+  "Stop translating %@": "Прервать перевод %@",
```
Comment (ForNeVeR) [inline app/lang/ru.json]: Would «Остановить» be better?

File: app/lang/ru.json (hunk 1 above)
Comment (raven-kg) [reply app/lang/ru.json]: Fixed

File: app/lang/ru.json (hunk 2 above)
Comment (raven-kg) [reply app/lang/ru.json]: I agree)

**Comment by bengotow** [comment]
Hey folks! Thanks for contributing these changes—they look great. (I don't speak Russian but I'll trust the two of you!) Will get these merged and shipped in the next release.

**Pull request merged**